index.html

```diff
@@ -0,0 +1,94 @@
+
+<!doctype html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+	<link  rel="stylesheet" type="text/css" href="style.css">
+	<title>Cubes</title>
+</head>
+<body>
+
+	<section class="wrapper">
+	
+		<div class="weather-widget">
+
+			<div class="weather-widget__card weather-widget__card_blue">
+				<img src="img\chicago.png" alt="chicago" />
+				<div class="widget__card_info">Chicago / 23°</div>
+			</div>
+
+			<div class="weather-widget__card weather-widget__card_green">
+				<img src="img\seatle.png" alt="seatle" />
+				<div class="widget__card_info">Seattle / 34°</div>
+			</div>
+
+			<div class="weather-widget__card weather-widget__card_darkblue">
+				<img src="img\brooklyn.png" alt="brooklyn" />
+				<div class="widget__card_info">Brooklyn / 55°</div>
+			</div>
+
+			<div class="weather-widget__card weather-widget__card_yellow">
+				<img src="img\portland.png" alt="portland" />
+				<div class="widget__card_info">Portland / 27°</div>
+			</div>
+
+			<div class="weather-widget__card weather-widget__card_red">
+
+				<div class="weather-widget__card_top">
+					<img src="img\submenu.png" alt="selected card" />
+				</div>
+
+				<div class="weather-widget__card_bottom">
+
+					<ul class="weather-widget__card__week">
+							<li class="card__day">
+								<span class="day__abbreviation">mon</span>
+								<div class="day__image-wrapper">
+									<img class="day__image" src="img\chicago.png" alt="monday"/>
+								</div>
+								<span class="day__temperature">22°</span>
+							</li>
+						
+							<li class="card__day">
+								<span class="day__abbreviation">tue</span>
+								<div class="day__image-wrapper">
+									<img class="day__image" src="img\chicago.png" alt="monday"/>
+								</div>
+								<span class="day__temperature">19°</span>
+							</li>
+
+							<li class="card__day">
+								<span class="day__abbreviation">wen</span>
+								<div class="day__image-wrapper">
+									<img class="day__image" src="img\san jose.png" alt="monday"/>
+								</div>
+								<span class="day__temperature">18°</span>
+							</li>
+
+							<li class="card__day">
+								<span class="day__abbreviation">thu</span>
+								<div class="day__image-wrapper">
+									<img class="day__image" src="img\brooklyn.png" alt="monday"/>
+								</div>
+								<span class="day__temperature">14°</span>
+							</li>
+					</ul>
+					
+				</div>
+			</div>
+
+			<div class="weather-widget__card weather-widget__card_purple">
+				<img src="img\san jose.png" alt="sanjose" />
+				<div class="widget__card_info fix">San jose / 47°</div>

```

Comment:
I would recommend renaming `fix` class to a BEM-modifier class instead, i.e. `card__info_san-jose` (please refer to my earlier comment for more information on BEM elements and modifiers).